frontend/app/views/custom_report_templates/_form.html.erb

```diff
@@ -10,10 +10,12 @@
 	<% record_types.delete('global') %>
 
 	<div id="template_information">
-		<h3><%= I18n.t "custom_report_template._frontend.section.template_information" %></h3>
-	
+		<h3><%= I18n.t "custom_report_template._frontend.section.template_information" %>
+		<%= link_to_help :topic => "accession_basic_information" %></h3>
```

Comment:
Is there a valid help topic to link this to now?  It def should be going to accession basic info.

Reply:
This is a placeholder just to get the question mark in there - we haven't created the manual page yet (though we will before custom reports are in a release).

Reply:
This is a blank manual page that will eventually have documentation for custom reports: https://archivesspace.atlassian.net/wiki/spaces/ArchivesSpaceUserManual/pages/2667839513/Creating+Custom+Reports+coming+for+ArchivesSpace+3.1.0. It's currently private, but the URL will stay the same, and Confluence provides redirects if the name changes.